setup.py

```diff
@@ -191,8 +192,13 @@ def can_import(module_name):
     Extension(
         "Bio.Cluster._cluster", ["Bio/Cluster/cluster.c", "Bio/Cluster/clustermodule.c"]
     ),
-    Extension("Bio.PDB.kdtrees", ["Bio/PDB/kdtrees.c"]),
     Extension("Bio.PDB.ccealign", ["Bio/PDB/ccealignmodule.c"]),
+    Extension("Bio.PDB.kdtrees", ["Bio/PDB/kdtrees.c"]),
+    Extension(
+        "Bio.PDB._bcif_helper",
+        ["Bio/PDB/bcifhelpermodule.c"],
+        include_dirs=[np.get_include()],
```

Comment:
Why do we need this (compared to other C code using numpy)?

Reply:
The other extension modules do not use NumPy's C API as far as I can tell. They don't have a `#include <numpy/...>` directive that I would expect in a module that uses the NumPy API.

Reply:
Can you avoid using the NumPy API by using the buffer protocol instead? That is how the other C extension modules in Biopython do it. See for example the C code in Bio.Cluster.

Reply:
Thanks for introducing me to the buffer protocol! I have updated by code to use the buffer protocol instead of NumPy's C API now.